src/pages/about-us/about-us.js

```diff
@@ -4,12 +4,33 @@ import { Link } from 'react-router-dom';
 import { useStyles } from './about-us.styles';
 import materialUiConstants from '../../configs/material-ui-constants';
 import { config } from '../../configs';
+import TableContainerRow from '../../containers/table-container-row';
+import TableContainerGenerator from '../../containers/table-container-generator';
+import { tableFooter, tableHeader, tableItems } from './operations/mockdata';
 
+const { aboutUsHeaderTitles } = config.tableHeadRowTitles;
+const { aboutUsTitles } = config.tableHeadRowTitles;
+const { aboutUsFooterTitles } = config.tableHeadRowTitles;
 const { ADD_ABOUT_US } = config.buttonTitles;
 const { pathToAboutUsAdd } = config.routes;
 
 const AboutUs = () => {
   const styles = useStyles();
+  const aboutUsHeaderItem = tableHeader.map(({ id, text }) => (
+    <TableContainerRow showAvatar={false} key={id} title={text} />
+  ));
+  const aboutUsItems = tableItems.map(({ id, title, text, image }) => (
+    <TableContainerRow
+      showAvatar={false}
```

Comment:
Regarding this avatar, there are some data about author on the mokaps under the author's avatars, will you add it?